subprojects/core/src/main/java/org/gradle/execution/plan/TaskInAnotherBuild.java

```diff
@@ -134,11 +135,6 @@ public boolean requiresMonitoring() {
         return true;
     }
 
-    @Override
-    public void require() {
-        // Ignore
-    }
```

Comment:
Not ignoring this one allows this foreign task to be included in the current build's task graph as it is computed.